.github/workflows/check-blocking-labels.yml

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+
+name: Check Blocking Labels
+
+on:
+  # run pipeline on pull request
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - labeled
+      - unlabeled
+  # run pipeline on merge queue
+  merge_group:
+  # run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-blocking-labels
+  cancel-in-progress: true
+
+jobs:
+  check-blocking-labels:
+    runs-on: ubuntu-latest
```

Comment:
If the CI is passing, does this mean that ubuntu-latest has finally migrated to 24.04? Then we can update it everywhere else, right?

Reply:
this check only does a simple shell-check. no Ubuntu 24.04 features (like a specific compiler version) are used. If this action were to be affected, then the entire GitHub is broken (shell incompatibility on the very simplest level) and I don't think a lot of people would like that. I think it's safe to assume that it's no issue here 😬